controllers/categoryController.js

```diff
@@ -1,3 +1,19 @@
-class categoryController {}
+const { Category } = require('../models');
+const httpStatus = require('../helpers/httpStatus');
+
+class categoryController {
+  static async getAllCategories(req, res) {
+   let categories = [];
+   try {
+    const results = await Category.findAll();
```

Comment:
Podrías elegir que campos traer con [attributes](https://sequelize.org/docs/v6/core-concepts/model-querying-basics/#specifying-attributes-for-select-queries)

Reply:
Hecho!